models/transaction.ts

```diff
@@ -28,7 +28,7 @@ export class Transaction {
   @ViewColumn({
     name: "acquirer_c"
   })
-  acquirer_descr: string;
+  acquirer_c: string; // TODO: It's an enum type
```

Comment:
For consistency with the other fields of the models can we remove the final character that indicates only the type? It's redundant information.
```suggestion
  acquirer: string; // TODO: It's an enum type
```

Reply:
You are right! Yesterday I not found a clean naming convention to make in evidence that they are all enums fields. Anyway if you think that is more clean we can remove _c

Reply:
Yes, is more clear and coherent with the `BPDCitizen` model/API where these suffixs are missing.

Reply:
done